qiita_pet/templates/sitebase.html

```diff
@@ -34,37 +34,20 @@
           jumbotron.style.visibility = "hidden";
         }
 
-        // from http://mrbool.com/how-to-detect-different-browsers-and-their-versions-using-javascript/25424
-        var objappVersion = navigator.appVersion;
-        var objAgent = navigator.userAgent;
-        var objbrowserName = navigator.appName;
-        var objfullVersion = ''+parseFloat(navigator.appVersion);
-        var objBrMajorVersion = parseInt(navigator.appVersion,10);
-        var objOffsetName,objOffsetVersion,ix;
-        // In Chrome
-        if ((objOffsetVersion=objAgent.indexOf("Chrome"))!=-1) {
-          objbrowserName = "Chrome";
-          objfullVersion = objAgent.substring(objOffsetVersion+7);
-        } // In Firefox
-        else if ((objOffsetVersion=objAgent.indexOf("Firefox"))!=-1) {
-          objbrowserName = "Firefox";
-        } // In Safari
-        else if ((objOffsetVersion=objAgent.indexOf("Safari"))!=-1) {
-          objbrowserName = "Safari";
-          objfullVersion = objAgent.substring(objOffsetVersion+7);
-          if ((objOffsetVersion=objAgent.indexOf("Version"))!=-1)
-            objfullVersion = objAgent.substring(objOffsetVersion+8);
-        }
-        // trimming the fullVersion string at semicolon/space if present
-        if ((ix=objfullVersion.indexOf(";"))!=-1)
-          objfullVersion=objfullVersion.substring(0,ix);
-        if ((ix=objfullVersion.indexOf(" "))!=-1)
-          objfullVersion=objfullVersion.substring(0,ix);
-        objBrMajorVersion = parseInt(''+objfullVersion,10);
-        if (isNaN(objBrMajorVersion)) {
-          objfullVersion = ''+parseFloat(navigator.appVersion);
-          objBrMajorVersion = parseInt(navigator.appVersion,10);
-        }
+        // from http://stackoverflow.com/questions/5916900/how-can-you-detect-the-version-of-a-browser
```

Comment:
Can you use the permanent link instead? http://stackoverflow.com/q/5916900/379593